src/pages/docs/src/codegen/search.codegen

```diff
@@ -13,6 +13,9 @@ module.exports = function() {
   });
 
   const allDocs = readdirSync(generated)
+    .filter(g => {
+      return g.includes('autogen');
```

Comment:
Can you explain that? I know you placed `autogenerated` on the source above, but `readdirSync` gives back the contents of the directory. Why only take some that have `autogen` in their name?

Reply:
Without this filter method, the `yarn build:docs` command will fail for me, because the bundler can not find a related file. Since the overview is not based on a markdown file like the other pages, I excluded it here.

Reply:
I think a better approach would be to avoid the filter here, but to check if `source` is non-empty in / before line 27.

That way instead of `'autogenerated'` we could make the source `undefined`, `null` or just `''`.

In this case instead of returning a document we return `undefined`, which we filter out as last step (via a simple `.filter(Boolean)`).

Reply:
adapted

- make source field in gen-overview-page.js undefined
- replaced filter method by checking if source is undefined, null or emptry